crates/types/src/config/worker.rs

```diff
@@ -594,3 +607,34 @@ impl SnapshotsOptions {
         super::data_dir("db-snapshots").join(partition_id.to_string())
     }
 }
+
+/// # Throttling options
+///
+/// Throttling options per invoker.
+#[derive(Debug, Clone, Serialize, Deserialize, derive_builder::Builder)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[cfg_attr(feature = "schemars", schemars(rename = "ThrottlingOptions", default))]
+#[builder(default)]
+#[serde(rename_all = "kebab-case")]
+pub struct ThrottlingOptions {
+    /// # Burst capacity
+    ///
+    /// The maximum number of tokens the bucket can hold.
+    /// Default: 1M
+    pub burst: NonZeroU32,
+
+    /// # Replenishment rate
+    ///
+    /// The rate in seconds at which the tokens are replenished.
+    /// Default: 100k
```

Comment:
The default here implies that `100k` is a valid input, it's not. The default here is confusing since throttling itself is optional.